src/test/java/uk/gov/hmcts/reform/sscs/helper/SscsHelperTest.java

```diff
@@ -0,0 +1,80 @@
+        sscsCaseData = SscsCaseData.builder().ccdCaseId("ccdId").build();
+        Appeal appeal = Appeal.builder()
+                .benefitType(BenefitType.builder().code(Benefit.CHILD_SUPPORT.getShortName()).build()).build();
+        sscsCaseData.setAppeal(appeal);
+    }
+
+    @Test
+    public void givenNoResponseDueDate_WhenOtherPartyIsAdded_ThenSetResponseDueDateTo14DaysInTheFuture() {
+        sscsCaseData.setDirectionDueDate(null);
+        sscsCaseData.setOtherParties(Arrays.asList(buildOtherParty("No", null), buildOtherParty("Yes", NO)));
+
+        updateDirectionDueDateByAnAmountOfDays(sscsCaseData);
+        assertThat(sscsCaseData.getDirectionDueDate()).isEqualTo(LocalDate.now().plusDays(14).toString());
+
+    }
+
+    @Test
+    public void givenResponseDueDateIsSet_WhenOtherPartyIsAdded_IfDueDateIsMoreThan14DaysOld_ThenDoNotUpdateDate() {
+        sscsCaseData.setDirectionDueDate(LocalDate.now().plusDays(21).toString());
+        sscsCaseData.setOtherParties(Arrays.asList(buildOtherParty("No", null), buildOtherParty("Yes", NO)));
+
+        updateDirectionDueDateByAnAmountOfDays(sscsCaseData);
+        assertThat(sscsCaseData.getDirectionDueDate()).isEqualTo(LocalDate.now().plusDays(21).toString());
+    }
+
+    @Test
+    public void givenResponseDueDateIsSet_WhenOtherPartyIsAdded_IfDueDateIsNotMoreThan14DaysOld_ThenReSetDueDateTo14DaysInTheFuture() {
+        sscsCaseData.setDirectionDueDate(LocalDate.now().plusDays(2).toString());
+        sscsCaseData.setOtherParties(Arrays.asList(buildOtherParty("No", null), buildOtherParty("Yes", NO)));
+
+        updateDirectionDueDateByAnAmountOfDays(sscsCaseData);
+        assertThat(sscsCaseData.getDirectionDueDate()).isEqualTo(LocalDate.now().plusDays(14).toString());
+    }
+
+    @Test
+    public void givenResponseDueDateIsSet_IfDueDateIsMoreThan14DaysOld_ThenDoNotUpdateDate() {
+        sscsCaseData.setDirectionDueDate(LocalDate.now().plusDays(21).toString());
+
+        updateDirectionDueDateByAnAmountOfDays(sscsCaseData);
+        assertThat(sscsCaseData.getDirectionDueDate()).isEqualTo(LocalDate.now().plusDays(21).toString());
+    }
+
+    @Test
+    public void givenResponseDueDateIsSet_IfDueDateIsNotMoreThan14DaysOld_ThenReSetDueDateTo14DaysInTheFuture() {
+        sscsCaseData.setDirectionDueDate(LocalDate.now().plusDays(2).toString());
+
+        updateDirectionDueDateByAnAmountOfDays(sscsCaseData);
+        assertThat(sscsCaseData.getDirectionDueDate()).isEqualTo(LocalDate.now().plusDays(14).toString());
+    }
+
+    private CcdValue<OtherParty> buildOtherParty(String wantsToAttend, YesNo confidentiality) {
+        return CcdValue.<OtherParty>builder().value(OtherParty.builder()
+                .sendNewOtherPartyNotification(YES)
+                .confidentialityRequired(confidentiality != null ? confidentiality : NO)
+                .hearingOptions(HearingOptions.builder().wantsToAttend(wantsToAttend).build())
+                .build()).build();
+    }
```

Comment:
I don't like this helper. Having a helper in a test is risky enough - how do you know it does what you want, do you need to test it? - but having a conditional within it makes it even worse.

I'd recommend extracting this out into the places it is used. Repetition like this is acceptable in a test because it is for clarity.

Since there are only two unique usages, each used 3 times, save them as global variables with descriptive names and set them up in the setUp. You may want to change it to BeforeEach as well.